libs/dh/market-participant/actors/feature-actors/src/lib/edit/dh-actors-edit-actor-modal.component.html

```diff
@@ -15,5 +15,44 @@
 limitations under the License.
 -->
 <ng-container *ngIf="actor">
-  <watt-modal [title]="actor.glnOrEicNumber"> </watt-modal>
+  <watt-modal
+    [loading]="isLoading"
+    [title]="actor.name"
+    *transloco="let t; read: 'marketParticipant.actorsOverview.edit'"
+  >
+    <form [formGroup]="actorForm" *rxLet="formInitialValues$">
+      <watt-form-field class="actor-field">
+        <watt-label>{{ t("name") }}</watt-label>
+        <input wattInput type="text" [formControl]="actorForm.controls.name" />
+        <watt-error>{{ t("required") }}</watt-error>
+      </watt-form-field>
```

Comment:
suggestion: a quick comment before doing a full review - it's recommended to use the "new" `WattTextFieldComponent` when working with inputs. [Docs in Storybook](https://main--61765fc47451ff003afe62ff.chromatic.com/?path=/docs/components-text-field--docs). The `watt-form-field` + `wattInput` combination will going to be removed soon.

Reply:
Fixed.